localization/pose_initializer/README.md

```diff
@@ -7,18 +7,41 @@ It receives roughly estimated initial pose from GNSS/user.
-| `/sensing/gnss/pose_with_covariance` | geometry_msgs::msg::PoseWithCovarianceStamped | pose from gnss         |
-| `/map/pointcloud_map`                | sensor_msgs::msg::PointCloud2                 | pointcloud map         |
+| Name                  | Type | Description                                                                              |
+| --------------------- | ---- | ---------------------------------------------------------------------------------------- |
+| `ndt_enabled`         | bool | If true, the pose will be estimated by NDT scan matcher, otherwise it is passed through. |
+| `stop_check_enabled`  | bool | If true, initialization is accepted only when the vehicle is stopped.                    |
+| `stop_check_duration` | bool | The duration used for the stop check above.                                              |
+| `gnss_enabled`        | bool | If true, use the GNSS pose when no pose is specified.                                    |
+| `gnss_pose_timeout`   | bool | The duration that the GNSS pose is valid.                                                |
 
-### Output topics
+### Services
 
-| Name             | Type                                          | Description                 |
-| ---------------- | --------------------------------------------- | --------------------------- |
-| `/initialpose3d` | geometry_msgs::msg::PoseWithCovarianceStamped | calculated initial ego pose |
+| Name                       | Type                                              | Description           |
+| -------------------------- | ------------------------------------------------- | --------------------- |
+| `/localization/initialize` | autoware_ad_api_msgs::srv::InitializeLocalization | initial pose from api |
+
+### Clients
+
+| Name                                         | Type                                                    | Description             |
+| -------------------------------------------- | ------------------------------------------------------- | ----------------------- |
+| `/localization/pose_estimator/ndt_align_srv` | tier4_localization_msgs::srv::PoseWithCovarianceStamped | pose estimation service |
+
+### Subscriptions
+
+| Name                                                        | Type                                          | Description          |
+| ----------------------------------------------------------- | --------------------------------------------- | -------------------- |
+| `/sensing/gnss/pose_with_covariance`                        | geometry_msgs::msg::PoseWithCovarianceStamped | pose from gnss       |
+| `/sensing/vehicle_velocity_converter/twist_with_covariance` | geometry_msgs::msg::TwistStamped              | twist for stop check |
+| `/map/pointcloud_map`                                       | sensor_msgs::msg::PointCloud2                 | pointcloud map       |
```

Comment:
I think `/map/pointcloud_map` is subscribed at `map_height_fitter`.
So, please move it to the readme for `map_height_fitter`.

Reply:
Done https://github.com/autowarefoundation/autoware.universe/pull/1500/commits/9a6e85fd322639749db272ff91c70347f6670d79.